src/main/resources/templates/home.html

```diff
@@ -4,9 +4,15 @@
 
 <head>
   <th:block th:insert="~{fragments/common :: head(title='')}"></th:block>
+  <link rel="stylesheet" th:href="@{'/css/welcome-message.css'}">
 </head>
 
 <body>
+<div id="welcome-message" class="welcome-message">
+  <h1>Welcome to Stirling PDF!</h1>
```

Comment:
Not sure if we want this as a popup or your current implementation 

Any UI changes need screenshots attached so we can see your change